client/state/selectors/is-following-reader-conversation.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { key } from 'state/reader/conversations/utils';
+import { CONVERSATION_FOLLOW_STATUS_FOLLOWING } from 'state/reader/conversations/follow-status';
+
+/*
+ * Get the conversation following status for a given post
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Object} params Params including siteId and postId
+ * @return {Boolean} Is the user following this conversation?
+ */
+export default function isFollowingReaderConversation( state, { siteId, postId } ) {
```

Comment:
food for thought but not for eating now:

we may eventually want to move to a selector like this:
```js
getReaderConversationFollowStatus = ( state, { siteId, postId } ) => ( {
  following: bool,
  muted: bool,
  not_followed,
} );

Reply:
You have have seen that I started out with a selector very similar to that :)

What I deduced was - the UI never needs to know whether a conversation has been muted. It just needs to know "are we following this, or not?" Previously I ended up doing logic like this inside the component:

```if ( followStatus === CONVERSATION_FOLLOW_STATUS_FOLLOWING) { ... }```

and I thought that was best wrapped up in the selector to simplify things.

Reply:
I stand corrected @samouri - we do potentially need a selector for fetching follow state for the `actionWithRevert` magic in the data layer. I'll look at refactoring this in a followup PR.